diff -ruN a/arch/x86/Kconfig.cpu b/arch/x86/Kconfig.cpu
--- a/arch/x86/Kconfig.cpu	2023-01-17 18:52:12.824003286 +0600
+++ b/arch/x86/Kconfig.cpu	2023-01-17 18:54:15.771982381 +0600
@@ -778,7 +778,8 @@
 	depends on IA32_FEAT_CTL && X86_FEATURE_NAMES
 
 menuconfig PROCESSOR_SELECT
-	bool "Supported processor vendors" if EXPERT
+	bool "Supported processor vendors"
+	default y
 	help
 	  This lets you choose what x86 vendor support code your kernel
 	  will include.
